### ISSUE TITLE
"The CRTs must be greater than the resolvedTs" fatal error found in puller

### ISSUE THREAD

**Issue by overvenus** [pingcap/ticdc#900]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

Add a random merge scheduler. TiCDC panics after a while.

```
[2020/08/31 16:02:12.739 +08:00] [FATAL] [puller.go:184] ["The CRTs must be greater than the resolvedTs"] [row="{\"OpType\":3,\"Key\":null,\"Value\":null,\"OldValue\":null,\"StartTs\":0,\"CRTs\":419131798999269410}"] [CRTs=419131798999269410] [resolvedTs=41913179927452057
8] [tableID=71] [stack="github.com/pingcap/log.Fatal
	github.com/pingcap/log@v0.0.0-20200511115504-543df19646ad/global.go:59
github.com/pingcap/ticdc/cdc/puller.(*pullerImpl).Run.func4.1
	github.com/pingcap/ticdc@/cdc/puller/puller.go:184
github.com/pingcap/ticdc/cd
c/puller.(*pullerImpl).Run.func4
	github.com/pingcap/ticdc@/cdc/puller/puller.go:220
golang.org/x/sync/errgroup.(*Group).Go.func1
	golang.org/x/sync@v0.0.0-20200625203802-6e8e738ad208/errgroup/errgroup.go:57"]
```

4. Versions of the cluster

   - TiCDC version (execute `cdc version`):

cca74ccec096b5f89cd0975d1b3b164011387b58 @#846

[cdc.log.tar.gz](https://github.com/pingcap/ticdc/files/5150129/cdc.log.tar.gz)

**Comment by overvenus** [comment]
[cdc.log.tar.gz](https://github.com/pingcap/ticdc/files/5166449/cdc.log.tar.gz)

**Comment by leoppro** [comment]
fixed in #927
